infrastructure/src/main/java/org/corfudb/infrastructure/ManagementAgent.java

```diff
@@ -94,6 +96,8 @@
     private Future failureDetectorFuture = null;
     private Future healingDetectorFuture = null;
     private boolean recovered = false;
+    private final int recoveryRetries = 3;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this final field static too. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1170)

Reply:
Done.